rflx/cli.py

```diff
@@ -64,6 +65,14 @@ def main(argv: List[str]) -> Union[int, str]:
         "files", metavar="FILE", type=str, nargs="+", help="specification file"
     )
     parser_graph.add_argument("-d", "--directory", help="output directory", default=".", type=str)
+    parser_graph.add_argument(
+        "--no-verification",
+        action="store_true",
+        help=(
+            "skip time-consuming verification of model"
+            " (CAUTION: this can lead to unexpected results)"
+        ),
+    )
```

Comment:
Do we want to keep this warning? I think the worst case is plotting a graph with invalid conditions and lengths, as the general structure of the graph is still verified.

Reply:
You're right, we should remove it. Displaying the unverified graph is legit (I use it from time to time to diagnose problems in the model). Removed in eb38976.